VultisigApp/VultisigApp/Chains/UTXOChainsHelper.swift

```diff
@@ -248,4 +253,91 @@ class UTXOChainsHelper {
             throw HelperError.runtimeError("fail to construct raw transaction,error: \(error.localizedDescription)")
         }
     }
+    func getUnsignedTransactionHex(keysignPayload: KeysignPayload) throws -> String {
+        let input = try getBitcoinSigningInput(keysignPayload: keysignPayload)
+        var plan: BitcoinTransactionPlan = AnySigner.plan(input: input, coin: coin)
+        
+        if coin == .zcash {
+            plan.branchID = Data(hexString: "5510e7c8")!
+        }
+        
+        // Build raw transaction manually using plan data
+        var rawTx = Data()
```

Comment:
why this one is dealing with raw bytes? can't we use the methods in wallet-core to construct a tx ?